src/constants.js

```diff
@@ -0,0 +1,11 @@
+const LOGIN_URL = 'https://api.storyblok.com/v1/users/login'
```

Comment:
Suggestion:
```
const API_URL = 'https://mapi.storyblok.com/v1/'
const LOGIN_URL = `${API_URL}users/login`
const SIGNUP_URL = `${API_URL}users/signup`
```

Reply:
Thanks for the suggestion, it's much better